unpin some dependencies to allow this to install in the pangeo env

ocefpaf opened the pull request:
@rsignell-usgs this should solve your with qhub and the pangeo env. I ran all the upstream tests against the pangeo env and things works as expected.

@rmcd-mscb it would be nice if we could loosen more pins upstream.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

rmcd-mscb commented:
@ocefpaf I committed a new version to pypi.  I removed the poetry.lock file from the repo, and added python 3.8 to the classifiers and to the python dependencies. I'm open to other suggestions.    That should come in soon.

ocefpaf commented:
> @ocefpaf I committed a new version to pypi. I removed the poetry.lock file from the repo, and added python 3.8 to the classifiers and to the python dependencies. I'm open to other suggestions. That should come in soon.

Let's try that first instead of this PR. I believe @rsignell-usgs can wait :-)
